website/docs/core-concepts/connecting-to-data-sources/connecting-to-databases.md

```diff
@@ -5,59 +5,75 @@ description: >-
-* [ArangoDB](../../reference/datasources/querying-arango-db.md)
-* [DynamoDB](../../reference/datasources/querying-dynamodb.md)
-* [ElasticSearch](../../reference/datasources/querying-elasticsearch.md)
-* [Firestore](../../reference/datasources/querying-firestore.md)
-* [Google Sheets](../../reference/datasources/querying-google-sheets.md)
-* [MongoDB](../../reference/datasources/querying-mongodb/)
-* [MS SQL](../../reference/datasources/querying-mssql.md)
-* [MySQL](../../reference/datasources/querying-mysql.md)
-* [PostgreSQL](../../reference/datasources/querying-postgres.md)
-* [Redis](../../reference/datasources/querying-redis.md)
-* [Redshift](../../reference/datasources/querying-redshift.md)
-* [Snowflake](../../reference/datasources/querying-snowflake-db.md)
-* [SMTP](../../reference/datasources/using-smtp.md)
+### Connecting to a database
 
-## Security
+<VideoEmbed host="youtube" videoId="sJIxtXInV14" title="How to connect to a datasource" caption="How to connect to a datasource | Example"/>
 
-Appsmith safely encrypts all your database credentials and stores them securely. Appsmith also does not store any data returned from your data sources and acts only as a proxy layer to orchestrate the API / Query calls. Since Appsmith is an open-source framework, you can [deploy it on-premise](../../getting-started/setup/), and audit it to ensure none of your data leaves your VPC.
 
-## Connecting to a Database
+:::tip
+Before connecting to a data source, you must whitelist the IP address of the Appsmith deployment on your database instance or VPC
 
-:::note
-Before connecting to a data source, you may need to whitelist the IP address of the Appsmith deployment on your database instance or VPC.
+18.223.74.85 and 3.131.104.27 are the IP addresses of the Appsmith cloud instances that need to be whitelisted
 
-**18.223.74.85** and **3.131.104.27** are the IP addresses of the Appsmith cloud instances that need to be whitelisted.
+This is a guide on how to whitelist appsmith on AWS.
 :::
 
-Once the appsmith instance has been [whitelisted](../../learning-and-resources/how-to-guides/aws-whitelist.md):
 
-1. Click on **+** next to **Datasources**.
-2. You’ll see a list of Datasources that Appsmith can connect to.
-3. Choose a Database from the [supported list](connecting-to-databases.md#supported-databases)
-4. Provide the configuration details required to connect to your database. You may need to contact your database admin to [whitelist appsmith cloud](../../learning-and-resources/how-to-guides/aws-whitelist.md)
-5. Click on **Test** to verify that Appsmith is able to connect to your database using the details provided by you.
-6. **Save** your **Datasources**.
 
-![](</img/create_ds.gif>)
+* On the **Explorer tab**, click the **+** sign next to **Datasources**. 
+* You’ll see a [list of Datasources](/reference/datasources/) that Appsmith can connect to.
+* Choose a **Database/API**.
```

Comment:
remove this . clubbed in one point above

Reply:
removed